src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -96,7 +96,7 @@ private void checkClash(Model model, int lessonIndex) throws CommandException {
         }
     }
     public static String getUsageInfo() {
-        return "\nUsage: addTask/task + [lesson index] [description]. "
```

Comment:
Please also be reminded to change the usage message in the failure output TempUG for addTask

Reply:
okay corrected it